auto_sizing/data/target_lists.toml

```diff
@@ -1,8 +1,8 @@
 [targets]
-locale = ["('EN-US')", "('EN-US', 'EN-CA', 'EN-GB')"]
-release_channel = ["release"]
-country = ["US", "GB"]
-user_type = ["new", "existing"]
+locale = ["EN-US", ["EN-US", "EN-CA", "EN-GB"]]
+normalized_channel = ["release"]
+country = ["US", "all"]
+user_type = ["new", "existing", "all"]
```

Comment:
Will this format affect exported data format? We need to make sure that format remains the same because the keys are used to determine/index the audience info on the experiment. The keys are built as a `:`-delimited string of the targeting values, e.g. `firefox_ios:release:['EN-CA','EN-GB','EN-US']:US`.

Reply:
Also if we change the format of this we should make sure the README reflects the update.

Reply:
When trying to refresh the manifest file, I got an error on `toml.load` -- I dont' think nested arrays are valid TOML, so we'll probably have to switch it back to a tuple.